omod/src/main/webapp/portlets/patientHeader.jsp

```diff
@@ -19,7 +19,15 @@
 
 <%-- Header showing preferred name, id, and treatment status --%>
 <div id="patientHeader" class="boxHeader${model.patientVariation}">
-<div id="patientHeaderPatientName"><c:out value="${model.patient.personName}" /></div>
+<div id="patientHeaderPatientName">
+<c:choose>
+<c:when test="${not fn:contains(pageContext.request.servletPath,'patientDashboardForm.jsp')}" >
```

Comment:
Since this is a portlet, I would think it's better to add a parameter/attribute to it that determines if the link should be displayed or not which defaults to false